ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -321,6 +321,29 @@ Rectangle {
         }
     }
 
+    function getLineStartPosition(selectionStart) {
+        const text = getPlainText()
+        const lastNewLinePos = text.lastIndexOf("\n\n", messageInputField.selectionStart)
+        return lastNewLinePos === -1 ? 0 : lastNewLinePos + 2
+    }
+
+    function prefixSelectedLine(prefix) {
+        const selectedLinePosition = getLineStartPosition(messageInputField.selectionStart)
+        insertInTextInput(selectedLinePosition, prefix)
+    }
+
+    function unprefixSelectedLine(prefix) {
```

Comment:
name of this function is slightly misleading because `prefix` actual content is not used at all. Line is just trimmed by `prefix.length` regardless of `prefix` content

Reply:
Hm - yes - I guess it is good to check if the line is actually starting with that prefix before trimming.

Reply:
or rename to sth like `trimLeftSelectedLine(count)` if checking is not needed

Reply:
sorry, I didn't noticed that you added condition here :)